src/components/NavigationTabs/index.js

```diff
@@ -125,6 +191,9 @@ function NavigationTabs({ location: { pathname }, history }) {
   useBodyKeyDown('ArrowRight', navigateRight)
   useBodyKeyDown('ArrowLeft', navigateLeft)
 
+  const providerMessage = showSaiHolderMessage && daiPoolTokenBalanceFormatted > 0 && onLiquidityPage
+  const generalMessage = showGeneralDaiMessage && daiBalanceFormatted > 0
```

Comment:
daiBalanceFormatted is a string. this check should be something like `showGeneralDaiMessage && !!daiBalance && !daiBalance.isZero()`, and same for the check right above this